docs/luigi-client-api.md

```diff
@@ -507,6 +507,9 @@ Navigates to the given path in the application hosted by Luigi. It contains eith
     -   `drawerSettings.backdrop` **[boolean](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Boolean)** By default, it is set to `false`. If it is set to `true` the rest of the screen has a backdrop.
     -   `drawerSettings.size` **(`"l"` \| `"m"` \| `"s"` \| `"xs"`)** size of the drawer (optional, default `"s"`)
 
+##### Return
```

Comment:
I'm not sure what needs to be documented here. Is "return" a new API function? If so, please update the API .js file with the documentation as well.

Reply:
Removed it from luigi client api